merge master into live-based

luyajun0205 opened the pull request.

openpublishbuild committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

openpublishbuild committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

openpublishbuild committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

openpublishbuild committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

openpublishbuild committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

openpublishbuild committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

Commit:
Update authenticate-azure-cli.md

Commit:
Merge pull request #20 from OPS-E2E-Prod/master

merge master to live-based

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

Commit:
Merge branch 'live-based' into master

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

Commit:
Merge branch 'master' into master-sync-work

Commit:
Merge pull request #24 from OPS-E2E-Prod/master-sync-work

Resolve syncing conflicts from master-sync-work to master

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

v-susu-zz commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[.openpublishing.redirection.json](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[2017-03-09-profile/docs-ref-autogen/TOC.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/network/traffic-manager/endpoint?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/account.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/account.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/account?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/account/lock.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/account/lock.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/account/lock?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/acr.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/acr.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/acr?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/acr/credential.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/acr/credential.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/acr/credential?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/acr/replication.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/acr/replication.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/acr/replication?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/acr/repository.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/acr/repository.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/acr/repository?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/acr/webhook.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/acr/webhook.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/acr/webhook?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/acs.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/acs.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/acs?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/acs/dcos.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/acs/dcos.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/acs/dcos?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/acs/kubernetes.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/acs/kubernetes.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/acs/kubernetes?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/group.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/group.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/group?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/group/member.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/group/member.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/group/member?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/sp.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/sp.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/sp?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/ad/user.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/ad/user.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/ad/user?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/advisor.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/advisor.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/advisor?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/advisor/configuration.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/advisor/configuration.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/advisor/configuration?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/advisor/recommendation.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/advisor/recommendation.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/advisor/recommendation?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/aks.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/aks.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/aks?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/appservice.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/appservice.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/appservice?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/appservice/plan.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/appservice/plan.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/appservice/plan?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/backup.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/backup.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/backup?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/backup/container.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/backup/container.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/backup/container?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/backup/item.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/backup/item.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/backup/item?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/backup/job.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/backup/job.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/backup/job?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/backup/policy.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/backup/policy.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/backup/policy?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/backup/protection.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/backup/protection.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/backup/protection?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/backup/recoverypoint.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/backup/recoverypoint.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/backup/recoverypoint?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/backup/restore.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/backup/restore.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/backup/restore?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/backup/restore/files.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/backup/restore/files.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/backup/restore/files?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/backup/vault.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/backup/vault.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/backup/vault?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/backup/vault/backup-properties.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/backup/vault/backup-properties.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/backup/vault/backup-properties?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/batch.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/batch.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/batch?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/batch/account.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/batch/account.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/batch/account?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/batch/account/autostorage-keys.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/batch/account/autostorage-keys.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/batch/account/autostorage-keys?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/batch/account/keys.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/batch/account/keys.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/batch/account/keys?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/batch/application.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/batch/application.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/batch/application?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/batch/application/package.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/batch/application/package.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/batch/application/package?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/batch/application/summary.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/batch/application/summary.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/batch/application/summary?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/batch/certificate.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/batch/certificate.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/batch/certificate?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/batch/job-schedule.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/batch/job-schedule.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/batch/job-schedule?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/batch/job.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/batch/job.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/batch/job?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/batch/job/all-statistics.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/batch/job/all-statistics.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/batch/job/all-statistics?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/batch/job/prep-release-status.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/batch/job/prep-release-status.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/batch/job/prep-release-status?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/batch/job/task-counts.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/batch/job/task-counts.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/batch/job/task-counts?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/batch/location.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/batch/location.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/batch/location?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |
[2017-03-09-profile/docs-ref-autogen/batch/location/quotas.yml](https://github.com/OPS-E2E-Prod/azure-docs-cli-python/blob/master/2017-03-09-profile/docs-ref-autogen/batch/location/quotas.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/e2eprod-azure-cli-docs/batch/location/quotas?view=azure-cli-2017-03-09-profile&branch=pr-en-us-32) (azure-cli-2017-03-09-profile) |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C19%5C0762c6d6-cb91-510a-10b9-a881d0bb3e83%5CPullRequest%5C201803190553380600-32%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=7ioR0b0ladMvtOpHaac5y9BhPFmmfVhHGQdh8kkwlxw%3D&st=2018-03-19T05%3A59%3A15Z&se=2018-04-19T06%3A04%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
Remove quotes around configuration settings. [Fixes #695]

sptramer committed:
Merge pull request #696 from sptramer/config/fix-quotes

Remove quotes around configuration settings. [Fixes #695]

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

Commit:
Update Document Content

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

derekbekoe committed:
Release notes 2.0.30

Commit:
Copyedit 2.0.30 release notes

sptramer committed:
Extensions/add docs (#692)

* Create extension subsection in ToC
* Add doc for the alias extension.
* Add CLI extensions list.
* Improve extension overview

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

sptramer committed:
Merge pull request #700 from derekbekoe/az-2.0.30

Release notes 2.0.30

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

mimig1 committed:
Removed instance of DocumentDB 

Looks like it's no longer used and we're scrubbing to remove this term

Commit:
Move CLI 1.0 install directions into this repo

sptramer committed:
Merge pull request #704 from sptramer/install/migrate-cli-1.0

Move CLI 1.0 install directions into this repo

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

azclibot committed:
Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=359633690
https://github.com/Azure/azure-cli-extensions/commit/448b457daecf2c90d11c0442c024a1ef4367461a

rloutlaw committed:
Merge pull request #703 from mimig1/patch-1

Removed instance of DocumentDB

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

sptramer committed:
Merge pull request #705 from Azure/az-auto-ext-doc-359633690

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

azclibot committed:
Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=361352282
https://github.com/Azure/azure-cli-extensions/commit/8cca699619480fc0dc4d6283e60e87b75bb191d9

sptramer committed:
Merge pull request #711 from Azure/az-auto-ext-doc-361352282

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

azclibot committed:
Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=361832942
https://github.com/Azure/azure-cli-extensions/commit/5adf06965c7fbe35bf4ecb8655d7b673c1a90c0f

sptramer committed:
Merge pull request #715 from Azure/az-auto-ext-doc-361832942

Update CLI extensions available doc

alexandair committed:
Update azure-cli-extension-alias.md

Fix typos.

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

alexandair committed:
Update azure-cli-extension-alias.md

Fix typos.

sptramer committed:
Merge pull request #717 from alexandair/patch-3

Update azure-cli-extension-alias.md

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

Commit:
Change CLI anchor links to be correct.

sptramer committed:
Merge pull request #721 from sptramer/ref/fix-links

Change CLI anchor links to be correct.

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

Commit:
Update Document Content

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)

azclibot committed:
Update CLI extensions available doc.
 Triggered by Azure/azure-cli-extensions - TRAVIS_BUILD_ID=363298464
https://github.com/Azure/azure-cli-extensions/commit/a0b24e846c3a9d74a5deddac5289753d5187869b

sptramer committed:
Merge pull request #726 from Azure/az-auto-ext-doc-363298464

Update CLI extensions available doc

fenxu committed:
Merging changes synced from https://github.com/Azure/azure-docs-cli-python (branch master)
